core/modules/system/css/system.admin.css

```diff
@@ -221,11 +199,13 @@ span.admin-missing {
   display: block;
   font-weight: normal;
 }
-.system-status-report .status-description {
+.system-status-report .status-summary .details-content-wrapper {
```

Comment:
Can we leave the `status-description` class? I wouldn't want themes that targeted that class specifically to stop working. We can remove our CSS but let's not change the markup.